scss/_accordion.scss

```diff
@@ -50,9 +50,9 @@
   letter-spacing: var(--#{$prefix}accordion-btn-letter-spacing); // Boosted mod
   background-color: var(--#{$prefix}accordion-btn-bg);
   border: 0;
+  outline-offset: add($outline-offset, var(--#{$prefix}accordion-border-width)); // Boosted mod
   @include border-radius(0);
   overflow-anchor: none;
-  @include transition(var(--#{$prefix}accordion-transition));
```

Comment:
Haven't checked all the PR yet but IMO you can let it here without actually being an issue. Otherwise you'd have to deprecate `--#{$prefix}accordion-transition` which is only used here, and add a "Boosted mod" comment.